aqi_monitor/apps/monitor/function_for_task.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime as DT, timedelta
```

Comment:
First of all, never name modules like "function for". Module's name has to represent the main aim of this module. So, please consider another name.

Reply:
Fixed